CONTRIBUTING_JS.md

```diff
@@ -135,6 +159,18 @@ A `gh-pages` branch will be created, and this should be published to via the Git
 
 This makes API docs available, and the types are linked through to from other modules published by aegir.
 
+aegir can also update the `About` section of a README.md based on the `@packageDocumentation` of your entry file.
+
+#### Front-end and UI
+
+##### React
+
+For projects utilizing React, adhere to the following standards:
+
+1. Component Naming: Use kebab-case for component names (e.g., `my-component`). This is different than idiomatic React, which uses PascalCase.​
+2. File Extensions: Use .tsx for files containing React components to leverage TypeScript's JSX support.​
+3. State Management: Prefer using React's built-in state management features. If additional state management libraries are needed, please ensure they are well-justified, align with the project's architecture, and are well-typed. 
```

Comment:
```suggestion
3. State Management: Prefer using React's built-in state management features. If additional state management libraries are needed, please ensure they are well-justified, align with the project's architecture, and are well-typed. (e.g. do not use redux-bundler)
4. do not use prop-types or `.defaultProps`
5. Use functional components, hooks, and context where appropriate. Classes may be used for the logic of your components, but the component itself should be functional.
```

Reply:
@achingbrain any thoughts on these additions?

Reply:
I don't have any strong preferences around react/redux/etc.  I don't use it much and every time I do, it seems to have changed completely so happy to go with whatever.